scripts/postprocessing/bcdi_bulk_surface_strain.py

```diff
@@ -19,9 +19,10 @@
 import bcdi.postprocessing.postprocessing_utils as pu
```

Comment:
This file seems based on an outdated version. All assert must be replaced by the raise SomeError (assert are removed during compilation, they should be used only in tests). Please revert your changes